msg/vehicle_command.msg

```diff
@@ -82,6 +82,12 @@ uint16 VEHICLE_CMD_PAYLOAD_PREPARE_DEPLOY = 30001	# Prepare a payload deployment
 uint16 VEHICLE_CMD_PAYLOAD_CONTROL_DEPLOY = 30002	# Control a pre-programmed payload deployment
 uint16 VEHICLE_CMD_FIXED_MAG_CAL_YAW = 42006            # Magnetometer calibration based on provided known yaw. This allows for fast calibration using WMM field tables in the vehicle, given only the known yaw of the vehicle. If Latitude and longitude are both zero then use the current vehicle location.
 
+
+# PX4 vehicle commands (beyond mavlink commands)
+uint32 VEHICLE_CMD_SET_GPS_GLOBAL_ORIGIN = 65536        # Sets the GPS co-ordinates of the vehicle local origin (0,0,0) position. |Empty|Empty|Empty|Empty|Latitude|Longitude|Altitude|
```

Comment:
TODO: we need to add a check for this in mavlink so these commands aren't forwarded.

Reply:
Done.